src/components/Stories/content.js

```diff
@@ -1,6 +1,234 @@
 import React from 'react'
+import chartImg from './../../assets/stories/platform_features/chart.png'
+import mvrvImg from './../../assets/stories/platform_features/mvrv.png'
+import mmaImg from './../../assets/stories/platform_features/mma.png'
+import distroImg from './../../assets/stories/platform_features/distro.png'
+import renTopsImg from './../../assets/stories/platform_features/ren.png'
+import btcTacImg from './../../assets/stories/platform_features/btc_ta.png'
+import weeklyUpdatesImg from './../../assets/stories/platform_features/weekly.png'
```

Comment:
please, host images [here](https://github.com/santiment/sanbase2/tree/master/assets/static/images/sanbase/Stories)

Reply:
@haritonasty , why does stories depend from images on other app?

Reply:
Wait backend appoves in their repo https://github.com/santiment/sanbase2/pull/1615

Reply:
for loading them by url

Reply:
But by import from app we also load them from url...

Reply:
Example from app: 
![image](https://user-images.githubusercontent.com/14061779/68944922-28551000-07c0-11ea-9ddb-a90d6ac3651d.png)

Reply:
And also strange imports from 'stage' application on production app
![image](https://user-images.githubusercontent.com/14061779/68945046-7b2ec780-07c0-11ea-8c17-40691f0f1252.png)

Reply:
But they will be in the bundle (maybe not in main, but will) and increase his size. Or I mess smth?

Reply:
Not in a bundle. For example you can see a direct link for logo 
https://app-stage.santiment.net/static/media/logo-sanbase.fcc9cb23.svg
It is imported in code as external file.

Reply:
ok, thanks. But for common illustrations (that will be use in different products) better to use 1 remote link